app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        Person.people[self.name] = self
+
+
+def create_person_list(people: list) -> list:  # List[dict] -> List[Person]
+    for person in people:
+        love_key = "husband" if "husband" in person else "wife"
```

Comment:
why you don't use `.get()` method? `person.get("h", "w")`?

Reply:
`person.get("husband")` returns the value under the key `-> person["husband"]`
I only want to get the key name `-> "husband" | "wife"`